scripts/build.sh

```diff
@@ -150,12 +202,15 @@ case "$product-$method-$platform" in
     ;;
 
   Firestore-xcodebuild-iOS)
+    set +u
```

Comment:
Don't do this `set +u` dance. Just add the sanitizer flags to `xcb_flags` so that we're always dealing with a non-empty array.

Yes this makes it possible to build Firebase with sanitizers, but unless someone changes the travis config to actually use it, there's no effective change.